### ISSUE TITLE
Team xkcd hats

### ISSUE THREAD

**Issue by jupiterio** [SilversApprentice/-ATC-X-Veni-Vidi-Vici#6]
General Discussion. @algmwc5, @Mrcomputer1, and I are the current members of the team.

**Comment by algmwc5** [comment]
For more "secret" stuffs let's talk in the slackroom with encrypted messages :stuck_out_tongue:

**Comment by jupiterio** [comment]
Well, there isn't much use in using encrypted messages if we're the only ones that are going to read the messages 😛

**Comment by CosmicWebServices** [comment]
Sure... You are the only one
-IcyCoder

**Comment by jupiterio** [comment]
@CosmicWebServices 
> **algmwc5:**
> ... in the slackroom ...

**Comment by CosmicWebServices** [comment]
Unless you are paying for slack I can still look at your messages...

**Comment by Mrcomputer1** [comment]
@CosmicWebServices 
In a free Slack room to see the messages you must be a member of the Slack room. Also there are Direct Messages and Private Channels in free Slack rooms. 

We are using a private channel in a free Slack room (scratch-aters.slack.com).

**Comment by CosmicWebServices** [comment]
sure private...

**Comment by CosmicWebServices** [comment]
:)

**Comment by CosmicWebServices** [comment]
OK so all you have to use is the slack api to get the messages

**Comment by CosmicWebServices** [comment]
look: https://api.slack.com/methods/channels.list

**Comment by Mrcomputer1** [comment]
But you an API token with permission to access scratch-aters

**Comment by CosmicWebServices** [comment]
> To retrieve a list of private channels, use groups.list.

**Comment by Mrcomputer1** [comment]
From the groups.list API method
>This method returns a list of private channels in the team **that the caller is in** and **archived groups that the caller was in**.

**Comment by Mrcomputer1** [comment]
You need the token of someone in the private channel.

**Comment by CosmicWebServices** [comment]
There are ways around that

**Comment by Mrcomputer1** [comment]
What ways?

**Comment by Mrcomputer1** [comment]
The ways probably break the Slack API Terms of Service or the Slack Terms of Service

**Comment by CosmicWebServices** [comment]
It does

**Comment by Mrcomputer1** [comment]
Then you can't use it to get the messages or you would get in big trouble with Slack if they find out

**Comment by CosmicWebServices** [comment]
Yeah all of this to say that private is not private at all (especially free plans on im services)

**Comment by Mrcomputer1** [comment]
Ok, so are information is sort of safe because:
A. No one would really want any information in the Slack room
B. Anyone that would try wouldn't have a use for any of the information in the private channel.

**Comment by CosmicWebServices** [comment]
A. I am competitive so yes I want to know ALL
B. That is what they all say....

**Comment by Mrcomputer1** [comment]
A: But you wouldn't because you would break the Slack API TOS
B: ...

**Comment by algmwc5** [comment]
Team xkcd hat members only please

**Comment by Mrcomputer1** [comment]
@algmwc5 

We have the slack private channel for private, team member only chat.

**Comment by CosmicWebServices** [comment]
OK I am unsubscribing...

**Comment by jupiterio** [comment]
Should log it here.

We have uploaded the first cracker for Saiid's encryption v1.0

**Comment by algmwc5** [comment]
Yay we are assigned

**Comment by SilversApprentice** [comment]
lol

**Comment by Mrcomputer1** [comment]
lol :P

**Comment by algmwc5** [comment]
@SilversApprentice your encryption might not be decryptable
Let's wait for @jupiterio for now

**Comment by jupiterio** [comment]
@algmwc5 ?

**Comment by algmwc5** [comment]
Welp, happy new year
I guess nobody won
